tests/model_serving/model_runtime/vllm_omni/constant.py

```
@@ -0,0 +1,124 @@
+    },
+}
+
+# Volumes/mounts shared with standard vLLM runtime
+OMNI_VOLUMES: list[dict[str, Any]] = PREDICT_RESOURCES["volumes"]
+OMNI_VOLUME_MOUNTS: list[dict[str, Any]] = PREDICT_RESOURCES["volume_mounts"]
+
+
+# Probe configuration — matches vllm-omni-cuda-runtime-template spec exactly.
+# Only fields explicitly set in the template are included; Kubernetes-defaulted
+# fields (e.g. timeoutSeconds=1, successThreshold=1) are intentionally omitted.
+# startup: max 1200 s window (40 × 30 s); no initialDelaySeconds on readiness/liveness
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)